types/api.ts

```diff
@@ -0,0 +1,10 @@
+export interface ApiChange {
+  version: string | string[];
+  'pr-url': string;
+  description: string;
+}
+
+export interface ApiUpdate {
+  type: 'added' | 'removed' | 'deprecated' | 'introduced_in' | 'napiVersion';
```

Comment:
nit: Shall we make the naming consistent? Maybe `introduced_in` -> `introducedIn`.
Do you know if it is dependent on what data source?

Reply:
I don't know, I just migrated the types from https://github.com/nodejs/nodejs.dev/blob/main/src/types/api.ts as is.

Reply:
yes `introducedIn`  is consistent

Reply:
It seems that the code is in this [file](https://github.com/nodejs/nodejs.dev/blob/main/util-node/apiDocsTransformUtils.js#L37-L53).
But I haven't actually understood how it works. 😅

Reply:
Ok

Reply:
Note, it should be `introduced_in`, the reason behind is that this comes from the source docs on https://github.com/nodejs/node, so in this case this should **not** be `introducedIn`.